src/components/AddCosignatoryInput/AddCosignatoryInputTs.ts

```diff
@@ -103,41 +103,12 @@ export class AddCosignatoryInputTs extends Vue {
   }
 
   /**
-   * Emits onAddCosignatory event when cosignatory is input as a public key
+   * Emits onAddCosignatory event when cosignatory is input as an address
    * @private
    * @return {void}
    */
   private async addCosignerFromAddress() {
-    const address = Address.createFromRawAddress(this.cosignatory)
```

Comment:
as discussed on slack, this check should not be removed as this _prevents_ end-users from creating invalid transactions.

If this check is not done, and users are allowed to create multisig modifications with unknown cosigners ("unknown" as in "unknown by the network"), those transactions will be *invalid* and the end-users will end up being banned which we really want to avoid.

I suggested to add **steps** and **information** on how to solve the "account must be known" problem.

Also, _converting an unknown account to msig_ shall be fine. Whilst _adding cosigners that are unknown accounts_ is not fine.

Reply:
As discussed on the daily since this is an existing problem with Public Key input as well, this will be handled with another issue( https://github.com/nemgrouplimited/symbol-desktop-wallet/issues/561 ), please exclude this concern from this review.

Reply:
This concern is not valid any more after the 0.96 release. Please see the comment in #561 for the details.